docs/intro.md

```diff
@@ -114,6 +114,20 @@ with COGReader(
     assert npy_tile.shape == (1, 256, 256)
 ```
 
-## More
+Learn more about the **NumpyTile** specification [here](https://github.com/planetlabs/numpytiles-spec).
+
+
+## `rio-tiler`'s magic: Partial reading
+
+When the output image size, or the AOI is smaller than the input image, `GDAL` will try to
+perform *decimated and/or spatial reads* on the raster source, minimizing the amount data to transfer. Because of this, performance will be optimal when the source format permits efficient partial reads.
+
+The Cloud Optimized GeoTIFF (COG) format is the **recommanded** format for rio-tiler because it's natively:
```

Comment:
```suggestion
The [Cloud-Optimized GeoTIFF (COG)](https://www.cogeo.org/) format is the **recommended** format for rio-tiler because it's natively:
```